provisioner/ansible/provisioner.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/hashicorp/packer/common"
 	"github.com/hashicorp/packer/common/adapter"
 	"github.com/hashicorp/packer/helper/config"
+	confighelper "github.com/hashicorp/packer/helper/config"
```

Comment:
ah funny, it is possible to import a package twice !? TIL

Reply:
oh oops.